pkg/apis/internal.acorn.io/v1/unmarshal.go

```diff
@@ -257,13 +257,15 @@ func impliedVolumesForContainer(app *AppSpec, containerName, sideCarName string,
 					// ignore error
 					continue
 				}
-				vSize, err := resource.ParseQuantity((string)(v.Size))
-				if err != nil {
-					// ignore error
-					continue
-				}
-				if existingSize.Cmp(vSize) < 0 {
-					existing.Size = v.Size
+				if v.Size != "" {
+					vSize, err := resource.ParseQuantity((string)(v.Size))
+					if err != nil {
+						// ignore error
+						continue
+					}
+					if existingSize.Cmp(vSize) < 0 {
+						existing.Size = v.Size
+					}
```

Comment:
Log or warn user when trying to shrink volume in else condition?

Reply:
This isn't shrinking the volume, it's growing it. Additionally this behavior isn't changing.

Reply:
Makes sense. Thought along the lines of:
```
if existingSize.Cmp(vSize) <= 0 {
    existing.Size = v.Size
    } else {
    log.Warn("Unable to shrink volume size")
    }
```